.github/workflows/build.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+
+# Usage:
+# ./.github/workflows/build.sh 
+
+# this script will modified upper directory istio-proxy repo
+# if self-used should attention that whether the upper directory have its own developing istio.proxy
+#first build should give permission to docker volumes
+#sudo chmod -R 777 /var/lib/docker/volumes
+
+set -e
+
+UPDATE_BRANCH=${UPDATE_BRANCH:-"master"}
+cd ..
+rm -rf istio-proxy
+git clone -b ${UPDATE_BRANCH} https://github.com/intel/istio-proxy.git
+cp -rf istio/ istio-proxy/ 
+cd istio-proxy
+
+# Update to release branch as intel/envoy has updated.
+./scripts/update_envoy.sh
+BUILD_WITH_CONTAINER=1 make build_envoy 
+BUILD_WITH_CONTAINER=1 make exportcache
+
+cd istio
+# export env
+TAG=${TAG:-"pre-build"}
+make build
+# replace upstream envoy with local envoy in build proxyv2 image
+cd  ..
```

Comment:
If use use subshells you don't need to do `cd ..`. It might make the flow simpler?

    (cd istio; make build)